Signal/translations/ar.lproj/Localizable.strings

```diff
@@ -2257,7 +2257,7 @@
 /* Info message embedding a {{time amount}}, see the *_TIME_AMOUNT strings for context. */
 "YOU_UPDATED_DISAPPEARING_MESSAGES_CONFIGURATION" = "لقد ضبطت وقت إخفاء الرسالة على %@.";
 
-// Strings Copied in from JSQMessagesViewController
+
```

Comment:
This change (and the other localization file changes outside of en.lproj) make me thing you edited them manually.

Generally we shouldn't be editing any translation files manually - did you update autogenstrings script appropriately?

I'm not actually worried about the change itself - it just seems like if you're making manual tweaks, they're going to be blown away the next time we run autogenstrings -> sync-translations since these are generated files.

Reply:
These were just some whitespace changes left behind by global search-and-replace.  I've reverted them.